Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -21,11 +21,42 @@
 #include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/SmilesParse/SmilesParse.h>
 
+#include <RDGeneral/BoostStartInclude.h>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <RDGeneral/BoostEndInclude.h>
+
 using namespace std;
 namespace RDKit {
 namespace MolStandardize {
 const CleanupParameters defaultCleanupParameters;
 
+#define PT_OPT_GET(opt) params.opt = pt.get(#opt, params.opt)
+void updateCleanupParamsFromJSON(CleanupParameters &params,
+                                 const std::string &json) {
+  if (json.empty()) {
+    return;
+  }
+  std::istringstream ss;
+  ss.str(json);
+  boost::property_tree::ptree pt;
+  boost::property_tree::read_json(ss, pt);
```

Comment:
What happens if the json is malformed?

Reply:
I would like to add a test for this, but otherwise it looks fine.

Reply:
I can add a test, and it would be interesting to know, but the JSON parsing is boost code.

Reply:
ok, so what happens with malformed input is that boost::property_tree throws a `json_parser_error` which is derived (eventually) from `std::runtime_error`.

On the Python side this ends up being translated to a `RuntimeError`:
```
In [3]: Draw.UpdateDrawerParamsFromJSON(d2d,"{foo:")
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-2a56b06ecb22> in <module>
----> 1 Draw.UpdateDrawerParamsFromJSON(d2d,"{foo:")

RuntimeError: <unspecified file>(1): expected key string
```
This doesn't seen unreasonable to me, but we could consider doing a PR to modify all of the RDKit code which uses boost::property_tree to throw a `ValueErrorException` instead.